app/views/topics/index.html.erb

```diff
@@ -47,8 +47,8 @@
 
 <div>
   <% if user_signed_in? %>
-    <span class="hidden-xs pull-right"><%= link_to t(:open_new_discussion, default: "Open Discussion"), new_topic_path, class: 'btn btn-primary' %></span>
+    <span class="hidden-xs pull-right"><%= link_to t(:start_discussion, default: "Start Discussion"), new_topic_path, class: 'btn btn-primary' %></span>
```

Comment:
Its actually not renamed, those are two different keys with slightly different semantics.  One of the things I tried to do here was not introduce new keys, which would then be unavailable in the other languages (or we would have to go chase down translations for.)  

I actually included the default text everywhere not to avoid crashing or exceptions- but as a matter of style- I think it makes the html more readable for people who may not be familiar with the project...